analyze.py

```diff
@@ -147,7 +147,7 @@ def main():
         rounds = reader.read().strip()
     plot(df, rounds, png)
 
-    print(f"Successfult. Files generated:\n  {csv}\n  {png}")
```

Comment:
This has been resolved on the main repository 3 days ago. https://github.com/niklas-heer/speed-comparison/commit/dac8c43b7ad5dda558cb4fe731cb91d6e0f4d478